eos-updater/poll-common.c

```diff
@@ -551,6 +551,7 @@ fetch_latest_commit (OstreeRepo *repo,
       g_clear_pointer (&ref, g_free);
       g_clear_pointer (&new_refspec, g_free);
       g_clear_pointer (&checksum, g_free);
+      g_clear_pointer (out_version, g_free);
```

Comment:
It’s a bit suspect to clear the `out_version` (what if the caller hadn’t initialised it to `NULL` before calling `fetch_latest_commit()`? — they don’t have to). It would be better to assign to a local `version` variable inside the loop, then steal that to `out_version` after the loop, just like is done for `out_checksum` and `out_new_refspec`.

Reply:
Pushed a fixup